files/zh-tw/web/http/index.md

```diff
@@ -33,24 +34,24 @@ slug: Web/HTTP
+- [HTTP 頭](/zh-TW/docs/Web/HTTP/Headers)
+  - : HTTP 消息頭用於描述資源或服務器或客戶端的行為。可以使用 `X-` 前綴添加自定義專有頭。其他的可以在 [IANA registry](https://www.iana.org/assignments/message-headers/message-headers.xhtml#perm-headers) 中找到，其原始定義在 [RFC 4229](https://tools.ietf.org/html/rfc4229)。 IANA 同時也維護著一份 [registry of proposed new HTTP message headers](https://www.iana.org/assignments/message-headers/message-headers.xhtml#prov-headers)。
 - [HTTP 請求方法](/zh-TW/docs/Web/HTTP/Methods)
-  - : 透過 HTTP 有幾種不同操作方法：{{HTTPMethod("GET")}}, {{HTTPMethod("POST")}}, and also less common requests like {{HTTPMethod("OPTIONS")}}, {{HTTPMethod("DELETE")}}, or {{HTTPMethod("TRACE")}}.
-- [HTTP 狀態回應碼](/zh-TW/docs/Web/HTTP/Response_codes)
-  - : HTTP response codes indicate whether a specific HTTP request has been successfully completed. Responses are grouped in five classes: informational responses, successful responses, redirections, client errors, and servers errors.
-- [CSP directives](/zh-TW/docs/Web/HTTP/Headers/Content-Security-Policy)
-  - : The {{HTTPHeader("Content-Security-Policy")}} response header fields allows web site administrators to control resources the user agent is allowed to load for a given page. With a few exceptions, policies mostly involve specifying server origins and script endpoints.
+  - : 透過 HTTP 有幾種不同操作方法：{{HTTPMethod("GET")}}, {{HTTPMethod("POST")}}，或是一些不太常見的請求方式，像是：{{HTTPMethod("OPTIONS")}}, {{HTTPMethod("DELETE")}}, or {{HTTPMethod("TRACE")}}.
+- [HTTP 狀態回應碼](/zh-TW/docs/Web/HTTP/Status)
+  - : HTTP 狀態碼用來表示特定的 HTTP 請求是否已成功完成。響應分為五類：消息響應，成功響應，重定向，客戶端錯誤和服務器錯誤。
+- [CSP 指令](/zh-TW/docs/Web/HTTP/Headers/Content-Security-Policy)
+  - : {{HTTPHeader("Content-Security-Policy")}} 響應報頭字段允許網站管理員控制頁面上哪些資源能夠被用戶代理程序加載。除了少數例外，此策略主要涉及指定服務器來源和腳本終端。
 
 ## 工具與資源
 
-Helpful tools and resources for understanding and debugging HTTP.
+有助於了解和調試 HTTP 的工具和資源。
 
-- [Firefox Developer Tools](/zh-TW/docs/Tools)
-  - : [Network monitor](/zh-TW/docs/Tools/Network_Monitor)
+- [Firefox 開發者工具](https://firefox-source-docs.mozilla.org/devtools-user/index.html)
+  - : [網絡監視器](https://firefox-source-docs.mozilla.org/devtools-user/network_monitor/index.html)
 - [Mozilla Observatory](https://observatory.mozilla.org/)
-  - : A project designed to help developers, system administrators, and security professionals configure their sites safely and securely.
+  - : 一個旨在幫助開發人員，系統管理員和安全專業人員安全地配置其站點的項目。
 - [RedBot](https://redbot.org/)
-  - : Tools to check your cache-related headers
-- [How Browsers Work](http://www.html5rocks.com/en/tutorials/internals/howbrowserswork/)
-  - : A very comprehensive article on browser internals and request flow through HTTP protocol. A MUST-READ for any web developer.
+  - : 用於檢查與緩存相關的 HTTP 頭的工具。
+- [瀏覽器的工作原理](https://web.dev/howbrowserswork/)
+  - : 一篇非常全面的關於瀏覽器內部實現與通過 HTTP 協議的請求流的文章。可以說是所有 Web 開發者的**必讀內容**。
```

Comment:
- : A very comprehensive article on browser internals and request flow through HTTP protocol. A MUST-READ for any web developer.
```suggestion
  - : 關於瀏覽器內部實作及 HTTP 通訊協定請求流程的一篇非常詳盡的文章。可以說是所有 Web 開發者都**必讀**的內容。
```